tests/baselines/reference/es5-asyncFunctionObjectLiterals.js

```diff
@@ -75,8 +75,8 @@ function objectLiteral1() {
             switch (_b.label) {
                 case 0:
                     _a = {
-                        a: y
-                    };
+                            a: y

```

Comment:
This indentation change seems unintentional, possibly from merging the computed property and `yield` indices in `visitObjectLiteralExpression`?

Reply:
I think the quickest way to address it would be to not set `EmitFlags.Indented` if there were no computed property names above.

Reply:
We set indented so the downlevel emit for computed properties looks like this:
```js
// ts
obj = { a: 1, [x]: 2 };

// js
obj = (_a = {
    a: 1,
  },
  _a[x] = 2,
  _a);
```

However when we are downleveling generators only, we don't need the indent:
```js
// ts
obj = { a: 1, b: yield z };
// js
...
switch (_b.label) {
  case 0:
    _a = {
      a: 1
    };
    return [4, z];
  case 1:
    obj = (_a.b = _b.sent(),
      _a);
  ...
}
```